src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -217,6 +221,40 @@ public function iLimitNumberOfItemsTo(int $limit): void
         $this->client->filter();
     }
 
+    /**
+     * @When I check :itemName data
+     */
+    public function iCheckData(string $itemName): void
+    {
+        /** @var string $lastResponseContent */
+        $lastResponseContent = $this->client->getLastResponse()->getContent();
+        /**
+         * @var array{
+         *      "@id": string,
+         *      "@type": string,
+         *      "variant": string,
+         *      "productName": string,
+         *      "id": int,
+         *      "quantity": int,
+         *      "unitPrice": int,
+         *      "originalUnitPrice": int,
+         *      "total": int,
+         *      "subtotal": int
```

Comment:
```suggestion
         *      "productName": string
```
Adding everything seems like an overkill